### PR TITLE
Adding info for CVE-2013-2134

### PR DESCRIPTION
**Pull request by cxd3796**

### CODE REVIEW

**Comment by JMdoubleU** [comment]
Can you double-check the announcement date? The Struts security bulletin that this fell under seems to have been published on 2013-06-03.

It might be helpful to include in the description that this issue is triggered by crafted requests containing OGNL code.

There's a lot of overlap between this and CVE-2013-2135 (curated by me!) as the fixes both fell under the same [Jira ticket](https://issues.apache.org/jira/browse/WW-4090). Are you able to understand the functional difference between these CVEs? It's not quite clear to me.

**Review by Chrestopher — COMMENTED** [review]
+4 upvotes, wildcard matching is interesting

File: cves/CVE-2013-2134.yml [hunk 1]
```diff
@@ -45,7 +45,9 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  By using wildcard matching (using a generic case that will match if a specific case is not provided),
```
Comment (JosefLenz) [inline cves/CVE-2013-2134.yml:49]: Sentences should still be readable when text in () is removed, this actually reads as "By using wildcard matching, an attacker can execute Java code that is not in the source code." I'd recommend rewording as its not clear exactly what the vulnerability is from this. Is Wildcard Matching a thing onto itself, or is the vulnerability from using a wildcard that matches a generic case?

Comment (cxd3796) [reply cves/CVE-2013-2134.yml:49]: To answer your question, wildcard matching is the latter.

**Pull request merged**